projects/tinkerbell/sandbox-local-registry/registry/registry_images.txt

```diff
@@ -0,0 +1,7 @@
+public.ecr.aws/l0g8r8j6/tinkerbell/tink/tink-worker:v0.6.0-eks-a-v0.0.0-dev-build.1627 tink-worker:latest
+public.ecr.aws/l0g8r8j6/tinkerbell/hub/image2disk:6c0f0d437bde2c836d90b000312c8b25fa1b65e1-eks-a-v0.0.0-dev-build.1536 image2disk:v1.0.0
+public.ecr.aws/l0g8r8j6/tinkerbell/hub/cexec:6c0f0d437bde2c836d90b000312c8b25fa1b65e1-eks-a-v0.0.0-dev-build.1536 cexec:v1.0.0
+public.ecr.aws/l0g8r8j6/tinkerbell/hub/writefile:6c0f0d437bde2c836d90b000312c8b25fa1b65e1-eks-a-v0.0.0-dev-build.1536 writefile:v1.0.0
+public.ecr.aws/l0g8r8j6/tinkerbell/hub/kexec:6c0f0d437bde2c836d90b000312c8b25fa1b65e1-eks-a-v0.0.0-dev-build.1536 kexec:v1.0.0
+public.ecr.aws/l0g8r8j6/tinkerbell/hub/oci2disk:6c0f0d437bde2c836d90b000312c8b25fa1b65e1-eks-a-v0.0.0-dev-build.1536 oci2disk:v1.0.0
```

Comment:
can you also update these to the newer tags?

Reply:
Did the commit id change as well or just the build number at the end of the tag?

Reply:
can you check ECR for the exact tags? Here's one for kexec for reference https://gallery.ecr.aws/l0g8r8j6/tinkerbell/hub/kexec